module/Decision/src/Decision/Model/SubDecision/Foundation.php

```diff
@@ -17,8 +17,11 @@ class Foundation extends SubDecision
 {
 
     const ORGAN_TYPE_COMMITTEE = 'committee';
-    const ORGAN_TYPE_AV_COMMITTEE = 'avc';
```

Comment:
In the gewisdb PR, this is left as is (as in its left at `ORGAN_TYPE_AV_COMMITTEE`). So either change it there as well, or do not change it here. But keep it consistent.

Reply:
Fixed this in the gewisdb PR